tests/licensedcode/data/plugin_license/text/scan.expected.json

```diff
@@ -1,4 +1,166 @@
+    },
+    {
+      "key": "linux-syscall-exception-gpl",
+      "short_name": "Linux Syscall Exception to GPL",
+      "name": "Linux Syscall Exception to GPL",
+      "category": "Copyleft Limited",
+      "owner": "Linux Foundation",
+      "is_builtin": true,
+      "is_exception": true,
+      "spdx_license_key": "Linux-syscall-note",
+      "other_urls": [
+        "http://www.gnu.org/licenses/gpl-2.0.txt",
+        "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/COPYING"
+      ],
+      "standard_notice": "NOTE! This copyright does *not* cover user programs that use kernel\nservices by normal system calls - this is merely considered normal use\nof the kernel, and does *not* fall under the heading of \"derived work\".\nAlso note that the GPL below is copyrighted by the Free Software\nFoundation, but the instance of code that it refers to (the Linux\nkernel) is copyrighted by me and others who actually wrote it.\nAlso note that the only valid version of the GPL as far as the kernel\nis concerned is _this_ particular version of the license (ie v2, not\nv2.2 or v3.x or whatever), unless explicitly otherwise stated.\nLinus Torvalds\nThis library is free software; you can redistribute it and/or modify it\nunder the terms of the GNU General Public License version 2 as published by\nthe Free Software Foundation.\nThis library is distributed in the hope that it will be useful, but WITHOUT\nANY WARRANTY; without even the implied warranty of MERCHANTABILITY or\nFITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for\nmore details.\nYou should have received a copy of the GNU General Public License along\nwith this library; see the file COPYING. If not, write to the Free Software\nFoundation, 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.\n",
+      "text": "NOTE! This copyright does *not* cover user programs that use kernel\nservices by normal system calls - this is merely considered normal use\nof the kernel, and does *not* fall under the heading of \"derived work\".\nAlso note that the GPL below is copyrighted by the Free Software\nFoundation, but the instance of code that it refers to (the Linux\nkernel) is copyrighted by me and others who actually wrote it.\n\nAlso note that the only valid version of the GPL as far as the kernel\nis concerned is _this_ particular version of the license (ie v2, not\nv2.2 or v3.x or whatever), unless explicitly otherwise stated.\n\nLinus Torvalds"
+    }
+  ],
+  "rule_references": [
+    {
+      "license_expression": "gpl-2.0 WITH linux-syscall-exception-gpl OR linux-openib",
```

Comment:
IMHO here we should just dump the rule, as is. The order should be that of https://github.com/nexB/scancode-toolkit/blob/a410f6ebb93a9b22343d7737897b61f3cd2a6ac3/src/licensedcode/models.py#L1773 .... but that order is not correct and should be updated to align exactly with the order of a dump, and the text should come last.